server/controllers/events/events_controller.go

```diff
@@ -49,15 +49,16 @@ const azuredevopsTestURL = "https://fabrikam.visualstudio.com/DefaultCollection/
 // VCSEventsController handles all webhook requests which signify 'events' in the
 // VCS host, ex. GitHub.
 type VCSEventsController struct {
-	CommandRunner  events.CommandRunner
-	PullCleaner    events.PullCleaner
-	Logger         logging.SimpleLogging
-	Scope          tally.Scope
-	Parser         events.EventParsing
-	CommentParser  events.CommentParsing
-	ApplyDisabled  bool
-	EmojiReaction  string
-	ExecutableName string
+	CommandRunner         events.CommandRunner
+	PullCleaner           events.PullCleaner
+	Logger                logging.SimpleLogging
+	Scope                 tally.Scope
+	Parser                events.EventParsing
+	CommentParser         events.CommentParsing
+	ApplyDisabled         bool
+	DisableEmojiReactions bool
+	EmojiReaction         string
+	ExecutableName        string
 	// GithubWebhookSecret is the secret added to this webhook via the GitHub
```

Comment:
Instead of disabling and having to maintain another flag, couldn't we simply fix the bug where the empty emoji isn't disabling the emoji reaction?

Reply:
I think we should definitely address the bug, which can be done as part of this PR. But with the upcoming work in #1084, I do think a global flag to disable emojis isn't a bad idea. Thoughts?

Reply:
Yes i suppose if we have multiple emojis then a disable flag makes more sense to me unless we want to disable by making each emoji config empty. I do like the flag idea better.

cc @jamengual @GenPage